common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -82,10 +79,10 @@ start_function() {
         err "$0: This command must be run as root"
         exit 1
     fi
-    [[ "$BWL_TYPE" == "DUMMY" ]] && platform_init_dummy
-    [[ "$BTL_TYPE" == "LOCAL_BUS" ]] && prplmesh_framework_init
-    [[ "$PRPLMESH_MODE" == "CA" || "$PRPLMESH_MODE" == "C" ]] && prplmesh_controller_start
-    [[ "$PRPLMESH_MODE" == "CA" || "$PRPLMESH_MODE" == "A" ]] && prplmesh_agent_start
+    [ "@BWL_TYPE@" = "DUMMY" ] && platform_init_dummy
+    [ "@BTL_TYPE@" = "LOCAL_BUS" ] && prplmesh_framework_init
+    [ "$PRPLMESH_MODE" = "CA" -o "$PRPLMESH_MODE" = "C" ] && prplmesh_controller_start
```

Comment:
Note that shellcheck advises in general to use:
```
[ "$PRPLMESH_MODE" = "CA" || [ "$PRPLMESH_MODE" = "C" ]
```
but TBH I don't understand why...